playground/eShopLite/AppHost/AppHost.csproj

```diff
@@ -7,6 +7,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <IsAspireHost>true</IsAspireHost>
     <AssemblyName>eShopLite.AppHost</AssemblyName>
+    <AspireDashboardDir>..\..\..\artifacts\bin\Aspire.Dashboard\Debug\net8.0\</AspireDashboardDir>
```

Comment:
Add TODO

Reply:
Moved this back up into the root `Directory.Build.props` (with a TODO).